gems/ibm_cloud_power/lib/ibm_cloud_power/version.rb

```diff
@@ -6,10 +6,10 @@
 The version of the OpenAPI document: 1.0.0
 Contact: kylej@us.ibm.com
 Generated by: https://openapi-generator.tech
-OpenAPI Generator version: 6.0.1
+OpenAPI Generator version: 6.2.1
 
 =end
 
 module IbmCloudPower
-  VERSION = '2.0.0'
+  VERSION = '1.0.0'
```

Comment:
This would be a version downgrade which we obviously can't have, we went to `2.0.0` due to a breaking change by @jaywcarman in 13c415444d0766a8b141c14a50b21ae581b01614
Are there any breaking changes here per the semantic version rules https://semver.org/ ?

If there are breaking changes we need to go to 3.0.0, otherwise 2.1.0 for a simple enhancement bump

Reply:
Okay, @alizapeikes can you drop the change to the version.rb file?  Typically this is done by the gem owner when releasing the gem.

I'm going to release a v2.1.0 and if we find something is breaking we can yank it and release it as v3.0.0

Reply:
> Okay, @alizapeikes can you drop the change to the version.rb file? Typically this is done by the gem owner when releasing the gem.
> 
> I'm going to release a v2.1.0 and if we find something is breaking we can yank it and release it as v3.0.0

@agrare am I dropping this change to version.rb altogether or am I updating it to `VERSION = '2.1.0'`?

Reply:
@alizapeikes you should drop the change to the version, I will update as part of the gem release process